05-Organizing-and-Executing-Simulations/04-Submitting-Cluster-Jobs.ipynb

```diff
@@ -0,0 +1,570 @@
+{
```

Comment:
<p>typo: cluster</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>I'm not seeing where the typo is.</p>